### PR TITLE
feat(Puppeteer/PlaywrightScraper): add support for headful crawling

### PR DESCRIPTION
**Pull request by vladfrangu**
Handles a long lasting request. I would love if you had any better descriptions for the option, as I don't fully know how to describe it past...just linking to an article explaining it

### CODE REVIEW

File: packages/actor-scraper/playwright-scraper/INPUT_SCHEMA.json [hunk 1]
```diff
@@ -118,6 +118,12 @@
             "groupCaption": "Browser masking",
             "groupDescription": "Settings that help mask as a real user and prevent scraper detection."
         },
+        "headful": {
+            "title": "Run browsers in headful mode",
+            "type": "boolean",
+            "description": "By default, browsers run in headless mode. You can toggle this on to run them in headful mode.",
```
Comment (metalwarrior665) [inline packages/actor-scraper/playwright-scraper/INPUT_SCHEMA.json:124]: I would just add a sentence as `Headful mode can help with certain rare anti-scraping protections but is slower and more costly.`

**Review by metalwarrior665 — CHANGES_REQUESTED** [review]
Just a small nitpick

**Comment by AndreyBykov** [comment]
I would maybe instead of naming it `headful` and disabling by default name it `headless` and enable it by default, as term `headful` is rather niche. It's used more now, but from my experience it's usually `headless` or `full` in discussions/docs, and e.g. playwright does not have `headful` in docs at all. But that's rather a nit.

**Comment by vladfrangu** [comment]
That makes more sense, you're right. I'll apply it :D

**Comment by AndreyBykov** [comment]
@vladfrangu let's see what others think - as we use it internally from the beginning of time =D

**Comment by vladfrangu** [comment]
> @vladfrangu let's see what others think - as we use it internally from the beginning of time =D

Welp, already pushed a commit 😆 If others agree with keeping it headless then yay, if not, just undo one commit and we're done :D

**Comment by metalwarrior665** [comment]
I would vote slightly more for headful as it is the thing you opt-in (and you don't have to think in double-negation) but not a big deal :)

**Review by B4nan — APPROVED** [review]

**Comment by vladfrangu** [comment]
Maybe we can make a poll or something internally about this? What do you think?

**Comment by metalwarrior665** [comment]
@vladfrangu It is not that important I think :) Just go with your gut :)

**Comment by vladfrangu** [comment]
I'm going to be merging this as is later today - to me it feels more accurate to name the option what it is (headless), defaulted to true, and it is also how users reading playwright/puppeteer docs would see it, but if you disagree let me know :D

**Pull request merged**